Sources/App/Views/PackageInfo.swift

```diff
@@ -16,13 +16,15 @@ struct PackageInfo {
     var title: String
     var description: String
     var url: String
+    var stars: Int
 }
 
 extension PackageInfo {
     init?(package: Package) {
         guard let repoName = package.repository?.name,
               let repoDescription = package.repository?.summary,
-              let repoOwner = package.repository?.owner
+              let repoOwner = package.repository?.owner,
+              let repoStars = package.repository?.stars
```

Comment:
This is an interesting one that @finestructure will have an opinion on. He just removed the `repoDescription` code from line 25 here as it was causing packages to disappear from the index if they did not have a description. I think this may now introduce that same problem for packages that we've not managed to get stars for?

Just like the change Sven made in #1239, we could just use 0 if stars does not have a value.

I'd love your input on this @finestructure

Reply:
Yes, we'd want to substitute in 0 here (as I see Sarah has already done now 👍)